README.md

```diff
@@ -64,3 +64,32 @@ Options:
 ```
 
 Previously known as [github.com/rakyll/boom](https://github.com/rakyll/boom).
+
+### Dynamic request body
```

Comment:
Maybe this should be disabled by default? So that no existing uses are broken
We could add an explicit flag to enable this

Reply:
i think it won't break